libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java

```diff
@@ -438,7 +443,14 @@ public void fasterRouteFound(DirectionsRoute directionsRoute) {
     }
   };*/
 
-  private Router.Callback routeEngineCallback = new NavigationViewRouteEngineListener(this);
```

Comment:
by removing this, should we also delete the class `NavigationViewRouteEngineListener`?

Reply:
good catch! done